ldes-server-domain/pom.xml

```diff
@@ -37,6 +37,13 @@
             <artifactId>jena-arq</artifactId>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/com.vividsolutions/jts -->
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <version>1.13</version>
```

Comment:
Is there a reason why we use this version ? latest version is 1.14 (https://mvnrepository.com/artifact/com.vividsolutions/jts-core/1.14.0)

Reply:
also extract version to root pom if possible 😸